diff --git a/dist/index.js b/dist/index.js
index 32f90026660ae588d2b1afb69210a267e4ea52fa..4dd44f64cf9e35ccac86a73a6e16c9843977fdf3 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -3,7 +3,7 @@ import {
 } from "./chunk-IVWNQO5P.js";
 
 // src/index.ts
-import semverCompare from "semver/functions/compare";
+import semverCompare from "semver/functions/compare.js";
 
 // src/shared/providers.ts
 var sqlProviders = [
